openlibrary/plugins/admin/code.py

```diff
@@ -389,7 +389,22 @@ def POST_block_account_and_revert(self, account):
             ]
             flattened_records: list[dict] = sum(added_records, [])
             keys_to_delete |= {r['key'] for r in flattened_records}
-            changeset_ids = [c.id for c in changes]
+
+            edition_or_work_keys = {}
+            for change in changes:
+                for item in change.changes:
+                    edition_or_work_keys[item.key] = change.id
+            response = web.ctx.site.get_many(list(edition_or_work_keys), raw=True)
```

Comment:
Drini to check if we can do a `web.ctx.site.things({'key': ['Key1', 'key2'], 'type': {'key': '/type/delete'}})`